src/index.html

```diff
@@ -14,7 +14,7 @@ <h1>LeaderBoard</h1>
     <section class="table-container">
       <div class="top-bar">
         <h2>Recent Scores</h2>
-        <button type="button" class="refresh-btn">Refresh</button>
+        <button type="button" id="refresh" class="refresh-btn">Refresh</button>
       </div>
       <div class="table" id="list">
         <!-- <ul class="list">
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 🚮 👍🏻